client/src/js/services/account.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+/**
+ * @description 
```

Comment:
Could you either remove these comments or fill them in?  Just placeholders for comments doesn't make any sense.